### DISCUSSION TITLE
The trained forces are much better than energy. How could I improve the results of energy?

### DISCUSSION THREAD

**Discussion by I10140317** [deepmodeling/deepmd-kit#3251] [General]
I have done some trainings on a small molecule and the obtained results is shown in the attached figure. 
The input.json is also attatched. 
As can be seen, the energy is much worse than force. I have tried to increase the start_pref_e and limit_pref_e as suggested in some discussions. However, both energy and force become worse than using the parameters given in the examples. 
I wonder whether there is any other suggestions to improve the results. 

Thanks!
[input.json](https://github.com/deepmodeling/deepmd-kit/files/14217366/input.json)
 
![ML](https://github.com/deepmodeling/deepmd-kit/assets/159509382/9c31c735-acf9-4fb2-9b87-6987415044fe)

**Comment by njzjz** [comment]
You may consider increase the prefactor of the energy loss, i.e. `loss/start_pref_e` and `loss/limit_pref_e`. The current value is quite small.

**Comment by njzjz** [comment]
By the way, you need to ensure the DFT accuracy is enough before doing
everything else.
